tests/McpResponse.test.ts

```diff
@@ -181,12 +181,9 @@ Call browser_handle_dialog to handle it before continuing.`,
         return [getMockRequest()];
       };
       const result = await response.handle('test', context);
-      assert.strictEqual(
-        result[0].text,
-        `# test response
-## Network requests
-http://example.com GET [pending]`,
-      );
+      const text = result[0].text as string;
+      assert.ok(text.includes(`## Network requests`));
```

Comment:
Let's revert to the previous assertion style to ensure relative order and verify full structure.